build/circle_tests.sh

```diff
@@ -3,4 +3,6 @@
 set -x
 set -e
 
+docker run -t --entrypoint="/usr/bin/run_unittests" -w "/root/src/preprocessing-workflow" oesteban/fmriprep
```

Comment:
You are almost there. Before the image name ```oesteban/fmriprep``` you need to specify the entrypoint. In your case it is ```--entrypoint=/usr/bin/run_unittests```. And you're done!